ISO8583/ISO8583-connector/ISO8583-connector-1.0.0/org.wso2.carbon.connector/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.carbon.connector</groupId>
+    <artifactId>org.wso2.carbon.connector.ISO8583</artifactId>
+    <version>4.2.0</version>
+    <packaging>jar</packaging>
+    <name>WSO2 Carbon - Mediation Library Connector For ISO8583</name>
+    <url>http://wso2.org</url>
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.mediation.initializer</artifactId>
+            <version>4.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.synapse</groupId>
```

Comment:
Get the generic dependencies from parent pom. Like synapse, carbon mediations.

Other version please use properties